src/main/java/org/folio/inventory/storage/external/ExternalStorageModuleHoldingsRecordCollection.java

```diff
@@ -1,22 +1,28 @@
 package org.folio.inventory.storage.external;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.JsonObject;
 import java.io.IOException;
-
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.folio.HoldingsRecord;
 import org.folio.dbschema.ObjectMapperTool;
 import org.folio.inventory.domain.HoldingsRecordCollection;
 import org.folio.inventory.validation.exceptions.JsonMappingException;
 
-import io.vertx.core.http.HttpClient;
-import io.vertx.core.json.JsonObject;
-
 class ExternalStorageModuleHoldingsRecordCollection
   extends ExternalStorageModuleCollection<HoldingsRecord>
   implements HoldingsRecordCollection {
 
   private static final Logger LOGGER = LogManager.getLogger(ExternalStorageModuleHoldingsRecordCollection.class);
+  private static final ObjectMapper mapper = ObjectMapperTool.getMapper();
+
+  // added to ignore unknown fields
```

Comment:
We override the regular mapping due to a specific reason - response from mod-inventory-storage has changed and cannot be mapped to a HoldingsRecord schema. Please list specific fields that we need to ignore. It is a workaround (we ignoring all unknown properties), and it should be acknowledged here